src/client/index.ts

```diff
@@ -157,7 +186,9 @@ export async function getAll(): Promise<Notification[]>{
 }
 
 /**
- * Clears all Notifications which were generated by your applications uuid.
+ * Clears all Notifications which were generated by your application's uuid.
```

Comment:
One more small one. Can we use similar text to `getAll` above (e.g. `Clears all Notifications which were created by the calling application, including child windows.`)